firefox_puppeteer/tests/test_page_info_window.py

```diff
@@ -69,6 +69,15 @@ def opener(win):
                            )
 
         for trigger in open_strategies:
+
+            if self.platform == 'winnt':
+                if trigger == 'shortcut':
+                    self.assertRaises(ValueError, self.browser.open_page_info_window,
+                                      trigger=trigger)
+                    continue
+                if callable(trigger):
```

Comment:
Change the opener to use the menu instead. We have to test that triggering via a callback works.